easybuild/tools/options.py

```diff
@@ -258,7 +258,7 @@ def override_options(self):
                                             None, 'store_true', False),
             'cleanup-builddir': ("Cleanup build dir after successful installation.", None, 'store_true', True),
             'cleanup-tmpdir': ("Cleanup tmp dir after successful run.", None, 'store_true', True),
-            'color': ("Allow color output", None, 'store_true', True),
+            'color': ("Colorize output", 'choice', 'store', 'auto', ['auto', 'always', 'never'], {'metavar':'WHEN'}),
```

Comment:
since enabling this requires having `humanfriendly` installed, I wouldn't enable this by default...

also, changing this may break people's configuration files, since they may specify `False` or `True` now, so we'll need to be careful with changing this to string, and at least translate `False` to `never` and `True` to `auto` (I wouldn't go with `always` since using that is probably rare...)